### PR TITLE
remove addRoutePattern call and sentence about it

### PR DESCRIPTION
**Pull request by aloupfor**

### CODE REVIEW

**Comment by aloupfor** [comment]
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1216
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/81

**Comment by ghost** [comment]
:+1:

File: en/developer_guide/backbone_routing.rst [hunk 1]
```diff
@@ -35,8 +32,7 @@ In a configuration file, you could add a route to the router:
         });
     })(window.appRouter);
 
-When you declare a new route in Backbone, you should first give the route pattern for the route generation.
-Then you should declare the method used for the matching.
+When you declare a new route in Backbone, you should declare the method used for the matching.
```
Comment (ngilain) [inline en/developer_guide/backbone_routing.rst]: I think this line became useless here at the moment the previous one was removed. You could remove it and add a modified one before the code sample to introduce it. Something like `To declare a new route in Backbone, you have to declare the method run when the route is triggered :`

**Comment by nicolasThal** [comment]
@itkg-ngilain doc updated

**Comment by ngilain** [comment]
:+1:

**Pull request merged**